src/Executor/Informer.php

```diff
@@ -28,6 +35,11 @@ class Informer
     public function __construct(OutputWatcher $output)
     {
         $this->output = $output;
+        
+        $deployer = Deployer::get();
+        if ($deployer->logs->has("email")) {
+            $this->sender = Deployer::get()->logs->get("email");
```

Comment:
You are getting `email` which is `sender` from `logs`. Better to separate concepts.